frame/support/procedural/src/construct_runtime/expand/call.rs

```diff
@@ -163,19 +163,6 @@ pub fn expand_outer_dispatch(
 				}
 			}
 		}
-		impl #scrate::traits::DispatchableWithStorageLayer for Call {
-			type Origin = Origin;
-			fn dispatch_with_storage_layer(self, origin: Origin) -> #scrate::dispatch::DispatchResultWithPostInfo {
-				#scrate::storage::with_storage_layer(|| {
-					#scrate::dispatch::Dispatchable::dispatch(self, origin)
-				})
-			}
-			fn dispatch_bypass_filter_with_storage_layer(self, origin: Origin) -> #scrate::dispatch::DispatchResultWithPostInfo {
-				#scrate::storage::with_storage_layer(|| {
-					#scrate::traits::UnfilteredDispatchable::dispatch_bypass_filter(self, origin)
-				})
-			}
-		}
```

Comment:
This needn't have been removed

Reply:
Was it removed in origin substrate as well?

Reply:
yes it was removed